src/sagemaker_pytorch_serving_container/torchserve.py

```diff
@@ -138,7 +141,7 @@ def _generate_ts_config_properties(handler_service):
     if ts_env.is_env_set() and not ENABLE_MULTI_MODEL:
         models_string = f'''{{\\
         "{DEFAULT_TS_MODEL_NAME}": {{\\
-            "1.0": {{\\
+            "1": {{\\
```

Comment:
curious why this change is made?

Reply:
This is a bug-fix in case this package is ever built with 1.9.1 container manually, there was a previous mismatch between the adapt_to_ts_format() method and this version number. That method has since been removed, and this version shouldn't matter in the newer implementation of the toolkit.